docs/en/observability/aws-elastic-serverless-forwarder.asciidoc

```diff
@@ -14,21 +14,21 @@ The Elastic Serverless Forwarder is an Amazon Web Services ({aws}) Lambda functi
 [discrete]
 [[aws-serverless-forwarder-overview]]
 = Overview
-The Elastic Serverless Forwarder can forward {aws} data to cloud-hosted or self-managed Elastic environments. It supports the following inputs:
+The Elastic Serverless Forwarder can forward {aws} data to cloud-hosted, self-managed Elastic environments or preview:[]{ls}. It supports the following inputs:
```

Comment:
```suggestion
The Elastic Serverless Forwarder can forward {aws} data to cloud-hosted, self-managed Elastic environments, or preview:[]{ls}. It supports the following inputs:
```
We use the oxford comma, though honestly...some folks have a hard time remembering.